homework1/index.css

```diff
@@ -0,0 +1,443 @@
+/*mobile*/
+@import url('https://fonts.googleapis.com/css2?family=Baloo+2:wght@400;600&display=swap');
+@media (min-width:320px) {
```

Comment:
Please, add a max-width query
And better to show the mobile view from 320px to 991px, it will help u to fix bugs with header in 801px - 916px screen resolution.
e.g. @media screen and (min-width: 320px) and (max-width: 991px)